### ISSUE TITLE
Missing results export icons on insiders build

### ISSUE THREAD

**Issue by kburtram** [microsoft/vscode-mssql#1436]
- MSSQL Extension Version: 1.7
- VSCode Version: Insider 10/17

Steps to Reproduce:

1. Run a query

Expected: 3 export icons

Actual: 1 export icon

Note: There are now also double scrollbars on the results document where there aren't on stable.

![image](https://user-images.githubusercontent.com/599935/67027580-a12c5380-f0be-11e9-8cf7-b6f37acfa32e.png)

**Comment by kburtram** [comment]
@abist I'm seeing this again now with the latest 1.7.1 RC build.  Are you able to repro?  I'm on Windows with the latest VS Code in case that matters.

**Comment by abist** [comment]
Waiting on VS Code team to repond to this - https://github.com/microsoft/vscode/issues/84543

**Comment by gdvallance** [comment]
This issue is present in non-insiders build too.

I raise the matter here so that there are not too many bug reports. However, would it be better to raise a separate ticket for a non-insider build?

The details of system etc is below: 
Version: 1.40.0 (system setup)
Commit: 86405ea23e3937316009fc27c9361deee66ffbf5
Date: 2019-11-06T17:02:13.381Z
Electron: 6.1.2
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.17134
